src/main/java/com/napier/sem/App.java

```diff
@@ -76,20 +76,20 @@ public static void main(String[] args) {
 
         // Use-case 5.1-5.3
         // Produce a report on all capital cities in the world organised by largest population to smallest
-         ArrayList<CapitalCity> capitalCities = dao.allCapitalCitiesIn(WORLD, "");
+//         ArrayList<CapitalCity> capitalCities = dao.allCapitalCitiesIn(WORLD, "");
```

Comment:
this isn't pretty. it was a mistake, if i make any other adjustments as a result of comments i'll fix it.